app/src/androidTest/java/com/android/sample/createRecipe/ListIngredientScreenTest.kt

```diff
@@ -177,4 +177,18 @@ class ListIngredientScreenTest {
     // Verify that the updateQuantity method was called with the correct parameters
     assertEquals(updatedQuantity, ingredientViewModel.ingredientList.value[0].quantity)
   }
+
+  @Test
+  fun testErrorDisplayedWhenNoIngredients() {
+    // Remove all ingredients from the list to simulate an empty state
+    mockedIngredients.forEach { ingredient -> ingredientViewModel.removeIngredient(ingredient) }
+
+    // Click the "Next Step" button
+    composeTestRule.onNodeWithTag(NEXT_STEP_BUTTON).performClick()
+
+    // Verify that the error message is displayed
+    composeTestRule
+        .onNodeWithText("Please add at least one ingredient to proceed.", useUnmergedTree = true)
```

Comment:
It would be better to use the xml constant you created for this text

Reply:
as the error message will be displayed through a toast and not directly on the screen this test will be deleted as it is deprecated